R/bibtex.R

```diff
@@ -196,10 +207,93 @@ findBibFile <- function(package) {
 #'
 #' @param file file name
 #' @param encoding encoding
-#' @param srcfile output of \code{\link{srcfile}}
+#' @param srcfile Deprecated
```

Comment:
Not needed nor used any more

Reply:
We should probably also modify the function signature to use `''` and write a quick diagnostic message: 

```r
  if (!missing("srcfile"))
    message("`srcfile` argument is deprecated.")
```

Reply:
Done! I also added a couple of tests on this